webapp/components/user_settings/user_settings_modal.jsx

```diff
@@ -113,12 +115,26 @@ class UserSettingsModal extends React.Component {
 
     handleKeyDown(e) {
         if (Utils.cmdOrCtrlPressed(e) && e.shiftKey && e.keyCode === Constants.KeyCodes.A) {
-            this.setState({
-                show: true
```

Comment:
Could have just replaced `true` with `!this.state.show` for a much simpler solution. 😛